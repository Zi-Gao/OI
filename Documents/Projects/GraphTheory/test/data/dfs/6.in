174 712
121 106 28202
8 114 29328
32 7 3470
107 134 18849
1 63 30516
72 87 6201
37 163 13376
26 25 22153
71 10 15774
47 29 30001
18 35 4083
52 10 26500
70 12 15685
76 129 14142
49 147 31351
56 149 30016
12 81 31463
173 73 30698
159 38 22227
48 111 8991
134 79 18774
149 136 16174
157 134 22807
84 65 31489
123 7 2811
144 18 27493
64 127 8321
37 133 31825
27 98 11150
123 125 10863
166 133 32723
35 80 19026
29 36 19987
51 61 28707
147 16 4083
53 21 1450
125 81 27416
92 19 5626
166 158 26819
91 159 7471
42 38 23340
42 140 4089
104 146 7234
57 109 8688
137 77 2466
121 112 25311
142 11 20451
119 137 27270
129 130 19335
65 39 18319
86 155 4141
71 4 4509
159 143 5856
153 107 10175
171 92 14995
102 173 7101
137 11 5672
40 143 24546
161 28 6549
123 149 20572
5 12 21513
65 84 29266
13 25 3863
162 60 4649
31 135 31577
30 72 6081
44 47 26881
122 110 17041
138 30 26454
28 13 22753
38 59 15383
6 92 31894
116 26 37
140 66 7505
39 129 24857
30 15 13008
9 89 8072
110 162 23499
38 30 28702
78 72 31939
25 106 31199
157 57 22411
77 83 22061
57 20 8035
39 135 18273
128 125 5890
110 152 24860
112 116 7397
57 66 10651
56 98 32699
87 150 23981
51 43 22472
121 169 2344
162 171 17292
58 21 9190
84 97 5410
102 171 26546
127 101 2817
57 110 11696
94 109 27182
11 128 25128
34 144 2309
13 45 5043
87 163 24853
35 172 24384
80 141 25554
39 157 3184
142 19 24829
128 74 9775
166 70 7962
5 133 18847
88 136 9551
8 20 22951
171 20 29557
12 166 20426
119 67 20556
47 107 4881
125 79 16141
61 147 9258
173 92 25810
49 142 6668
155 43 31624
144 85 27841
129 37 13163
144 102 29733
41 11 25910
135 132 24248
31 1 1686
130 153 18647
63 91 16251
86 22 5882
38 153 6745
47 152 21957
163 17 25214
36 10 14608
76 113 11014
3 50 28469
151 37 32274
4 67 28734
47 102 10060
140 37 18640
18 120 15844
74 137 11999
141 110 20866
58 86 663
66 131 15078
29 101 18429
26 158 23599
108 124 13865
27 83 26106
3 135 16180
91 40 19932
112 72 51
46 167 22498
132 87 27336
46 47 14365
146 46 16951
142 102 12903
156 13 17412
70 13 3549
145 54 6272
137 50 32456
163 15 468
12 65 9449
65 110 16484
167 147 13669
22 144 22949
151 49 11967
129 125 27324
2 41 23211
91 155 636
49 41 13464
63 67 13985
17 24 23874
66 128 22101
75 148 25267
129 72 20584
69 173 8565
80 142 11710
167 28 12003
172 130 28895
42 18 26081
98 133 20915
30 132 9819
87 135 14536
52 165 28000
50 109 8123
2 23 28344
163 85 9365
75 92 6711
138 76 15426
49 55 17999
129 159 5221
156 157 26239
160 94 28473
77 122 4024
117 24 9279
111 162 26065
132 152 1771
53 64 13466
17 43 20621
109 162 15413
101 98 23842
10 135 27576
83 101 8923
6 90 13899
126 108 23162
19 72 11835
11 53 25296
119 161 28346
33 38 5124
99 11 26689
14 122 1900
64 90 11480
125 10 24350
105 35 12455
138 125 25731
141 76 26779
16 142 30609
168 24 5138
116 143 20601
121 52 8127
169 59 5099
38 168 16195
33 59 15935
122 97 26680
81 160 1054
120 51 5276
147 68 1933
145 81 27610
151 32 13914
55 61 30957
46 60 25440
104 52 14662
92 133 15911
93 100 25475
68 89 17590
42 153 5655
81 144 14789
19 40 32114
170 35 7561
8 145 15949
54 129 12744
144 129 23585
24 107 16904
41 62 15609
114 39 20206
166 64 8983
163 7 26652
25 107 18986
103 49 579
155 146 6404
133 163 32538
13 102 23062
170 71 31607
121 104 31763
137 172 23492
101 72 18828
117 125 2430
61 101 8558
97 52 29830
145 74 21401
71 91 16745
3 4 11280
87 57 10734
98 56 15480
10 165 1923
50 67 26322
57 102 22379
84 65 23999
45 80 22191
131 161 3504
40 125 10313
34 130 32711
156 105 25089
64 25 4699
142 114 27884
163 61 11355
91 172 27802
15 103 10244
143 97 27723
70 43 26277
55 119 29640
96 146 8806
161 172 28080
76 133 29780
168 148 1885
54 120 29717
109 22 30382
38 137 9234
69 16 27727
26 142 22533
138 83 31074
99 153 15652
166 138 26000
101 114 29277
123 19 28397
82 91 1122
59 66 16175
150 139 29821
80 122 6704
173 32 602
133 136 30640
50 46 4172
142 86 28362
41 119 30399
133 115 9127
173 40 18119
19 154 22606
146 15 9495
162 147 26523
54 94 24006
68 9 24009
5 44 26276
162 158 9241
153 90 2320
107 107 23923
24 82 11666
85 138 12310
129 11 4770
111 59 23731
153 169 25954
29 34 15753
38 121 3643
62 73 10329
142 61 23702
55 75 17481
87 83 25643
49 165 16662
77 111 9117
124 99 3875
3 144 16220
27 82 20543
74 31 3328
46 140 18239
111 52 10912
75 162 8386
97 110 15053
143 3 6580
165 104 26940
135 75 22632
22 3 25924
50 70 21352
46 95 22197
149 88 27980
105 162 18112
36 121 3885
6 84 23002
151 29 5019
64 169 22759
65 89 21017
63 109 14098
5 145 8077
98 42 20244
16 14 12838
11 155 19942
1 124 6051
101 38 31236
55 39 25353
96 34 3537
12 167 29474
129 25 192
67 120 17750
15 119 11846
124 27 16590
84 85 6858
10 80 21460
109 34 30210
37 77 12779
29 55 14218
154 3 2846
113 113 7572
63 93 17777
7 149 16122
75 68 10106
10 162 6441
163 54 3982
10 61 22416
1 39 21757
164 66 21915
156 151 22880
24 152 23915
104 75 11743
7 6 13500
16 42 5363
105 5 30401
94 68 8987
104 163 21727
147 78 24952
61 94 29064
103 15 11083
171 117 28584
35 103 7111
27 83 22115
91 57 21441
72 108 23573
111 166 14103
56 124 23759
149 9 4073
170 151 32715
67 5 23036
130 49 11405
93 44 25309
18 173 15269
31 162 31648
12 76 6225
137 128 16201
89 83 8057
29 43 32069
79 60 15788
30 126 14721
166 114 28973
47 48 31738
92 172 13879
71 86 26592
59 6 21242
133 163 22172
84 125 17920
110 43 3427
163 61 5374
66 140 9232
108 115 13113
173 123 27269
136 81 32264
81 25 5567
144 115 13086
30 24 20572
22 16 6047
136 60 593
23 93 6183
142 1 20979
154 26 18771
142 152 5292
97 138 11630
144 97 12266
88 121 20295
83 113 23343
95 136 27490
80 85 23116
59 59 22527
67 79 13099
4 19 29101
70 150 9392
68 136 21862
127 169 19618
9 110 29282
8 3 2195
126 124 19746
85 160 16962
108 54 31338
141 34 4595
33 16 13339
11 147 419
24 63 17200
143 158 21039
170 81 8520
24 46 32704
80 51 31114
59 75 29982
163 33 25533
30 52 30132
94 85 9706
52 19 21613
117 134 18418
50 136 8629
3 21 8646
166 2 9533
112 35 6228
22 34 18517
77 44 17988
123 142 7164
24 21 9489
13 160 29174
158 6 16426
98 134 8835
61 10 24351
32 21 8334
116 100 13572
102 46 31856
128 12 21133
25 32 31132
37 158 9622
38 115 30150
142 158 3086
20 60 8298
120 91 14723
132 8 30658
129 153 13762
24 105 15492
117 158 24638
55 90 8191
100 36 30535
69 81 26743
113 99 29489
10 119 13773
125 102 25855
104 23 1230
147 83 17708
40 57 6605
57 102 2723
57 35 11082
99 74 19216
111 116 3024
70 143 23816
38 141 15968
145 146 18875
105 133 25329
134 135 7983
126 88 20727
67 148 13621
121 36 5105
4 4 1683
151 86 25186
59 78 11548
123 149 8658
24 5 10926
94 97 10907
60 82 16972
21 142 21686
52 107 3020
46 146 29377
43 13 29139
23 135 21240
61 45 15636
76 5 13780
99 80 275
51 134 30859
124 133 32211
147 51 1455
127 148 10040
14 116 2195
164 166 19979
46 62 28741
67 137 12252
173 90 5441
58 53 19003
145 111 24864
4 42 761
38 51 14835
62 15 23152
67 111 15126
144 34 15886
52 104 907
21 70 140
21 109 9804
15 104 19092
15 121 8975
37 39 20948
19 78 20080
96 141 7587
146 83 18050
173 24 26559
32 71 3377
121 111 3295
57 147 13313
165 82 21502
2 72 15618
82 161 25769
74 72 800
10 39 25542
67 6 12304
19 3 11139
44 114 4230
43 41 15973
15 47 1920
105 11 16066
89 66 1990
137 30 11012
40 102 26349
83 160 13434
104 28 8064
70 124 2488
100 159 16417
124 133 11942
22 124 25529
82 99 19407
29 64 13074
6 104 2967
10 17 10870
93 93 30552
165 9 15903
46 73 8094
12 40 3742
148 101 499
41 60 14987
49 49 25596
30 102 3345
122 6 3803
142 172 24889
133 166 18186
86 95 8196
109 97 23600
116 160 21417
96 156 2468
73 130 29855
143 93 27529
90 110 25906
100 47 21978
59 141 5848
30 61 12729
135 166 8321
59 21 31449
82 153 12177
140 109 23008
19 7 17746
29 4 31254
50 66 12528
171 105 26549
99 148 14956
129 12 20924
147 143 3327
97 1 17682
168 122 12258
4 75 12375
101 67 21124
1 5 1728
5 119 28039
3 144 32194
39 18 19489
12 16 29828
164 67 2481
52 79 25934
163 152 32174
136 21 8297
63 126 1074
102 102 19763
61 72 7125
78 108 6479
5 144 404
117 117 26426
150 96 23023
162 80 1203
159 123 8851
48 115 18091
49 100 10782
114 141 11200
21 134 13279
166 15 10100
62 133 19371
50 119 18326
102 112 12818
120 66 21147
102 89 17313
95 16 6362
76 149 24047
12 149 19430
63 110 7081
81 49 5758
109 43 23773
78 87 13512
105 28 808
28 114 16337
71 128 8611
137 62 4237
109 40 11927
104 129 24009
112 100 6500
149 26 21960
167 143 4385
113 47 16270
51 131 22748
27 6 30972
120 142 19586
44 117 15946
131 129 17883
156 58 23690
100 143 29470
6 14 5168
23 74 30978
124 21 20954
20 136 30816
154 161 21194
155 85 21437
91 121 6093
29 142 27477
166 86 1159
136 110 1664
6 67 609
6 104 22408
129 116 17796
33 14 837
150 145 14245
161 93 26655
7 55 24473
145 47 6213
98 51 11744
43 140 10550
75 138 13735
166 42 12021
65 77 22268
48 59 891
141 14 2592
56 168 27459
49 166 9528
77 144 19978
135 80 27862
116 138 10001
106 116 21773
112 42 4746
37 146 23560
113 166 1522
18 37 10336
168 15 24455
35 132 25769
152 127 7459
104 163 32489
126 110 17682
76 122 19955
144 78 27596
111 27 30690
173 80 22423
10 1 7211
