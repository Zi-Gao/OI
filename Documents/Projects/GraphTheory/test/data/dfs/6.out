1 5 12 16 14 116 26 25 32 7 6 67 79 60 82 91 40 57 20 136 21 70 13 45 80 51 43 41 11 53 64 90 110 152 127 101 38 30 15 47 29 4 19 3 50 46 62 73 130 49 55 39 18 35 103 132 8 114 141 34 144 78 72 87 83 113 99 74 31 135 75 68 9 89 66 128 125 10 17 24 63 93 44 117 134 158 100 36 121 52 104 23 28 129 37 77 111 27 98 42 140 109 22 124 133 115 163 33 59 54 94 85 138 76 122 97 149 88 160 120 142 61 147 143 86 95 155 146 102 112 171 92 172 105 162 173 123 166 2 153 107 169 56 168 148 159 165 106 126 108 119 137 161 81 131 150 96 156 58 151 157 139 145 167 154 48 84 65 71 